src/test/java/com/bitso/bitsoLedgerEntryTest.json

```diff
@@ -0,0 +1,3 @@
+{"success":true,"payload":[]}
```

Comment:
This file shouldn't live in the java code file path. Would be best to add a top-level folder with these test cases.